data/presets/highway/crossing/signals-raised.json

```diff
@@ -0,0 +1,32 @@
+{
+    "icon": "temaki-railway_signals",
+    "fields": [
+        "crossing",
+        "tactile_paving",
+        "crossing/island"
+    ],
+    "geometry": [
+        "vertex"
+    ],
+    "tags": {
+        "highway": "crossing",
+        "crossing": "traffic_signals",
+        "traffic_calming": "table"
+    },
+    "reference": {
+        "key": "traffic_calming",
+        "value": "table"
+    },
+    "name": "Pedestrian Signals Crossing (Raised)",
```

Comment:
It was added for the first time in https://github.com/openstreetmap/iD/commit/21a090ba26d71a8818720e4d5aeb5d397783631b#diff-e36bcdacdd8b73a0deab5086307b4b7acb91882eead46f6ff2a21150b81b991f AFAICS. So maybe @bhousel can answer this?

At first glance I would agree that the presence of a `traffic_calming=table` should be modeled in a field, not a separate preset.

Reply:
I think these existed much earlier when the `traffic_calming` presets were added:
https://github.com/openstreetmap/iD/commit/32b67e8c2c8898dbd307175d2d4b597447d0a0ab

There was a discussion here too: https://github.com/openstreetmap/iD/issues/4309

So I guess `traffic_calming=table` existed for a long time as its own preset, then I made the other presets for whether that feature was combined with a marked crosswalk or not. 

I don't know whether we could add a checkbox field that adds `traffic_calming=table` (checkboxes mostly just do `yes`/`no`)

Another reason to create it this way is that a preset that sets multiple tags will have a higher match score when all of those tags match.  I don't know whether it matters in this case, but I could imagine a situation where we consider "raised" as an attribute of the crosswalk, the user clicks the "raised" field, and then iD matches the traffic calming preset instead of the crosswalk preset.  (we can work around that behavior with match scores - we see it with buildings too - but it is a thing to know).

Reply:
FYI: I've converted these to a field in https://github.com/openstreetmap/id-tagging-schema/commit/4644b468ac5a9c1fc3d1521792e4265feb5a21b4. It seems to work just fine.